cloudvolume/cloudvolume.py

```diff
@@ -14,6 +15,7 @@
 from tqdm import tqdm
 from six import string_types
 import multiprocessing as mp
+from time import strftime
```

Comment:
Is this actually being used?

Reply:
yes, in the provenance generation:
https://github.com/seung-lab/cloud-volume/pull/188/files/cb58e847324315c7dbd5e84b44395df60181ff3c..419da54ecde51217d61f75675fd20c0031b88d21#diff-7095eb78d90e99a1cebf3f6e0c41a094R190